StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetError.swift

```diff
@@ -69,60 +69,60 @@ extension PaymentSheetError: CustomDebugStringConvertible {
     /// A string that can safley be logged to our analytics service that does not contain any PII
     public var safeLoggingString: String {
```

Comment:
We have to construct these case names manually since we override `debugDescription` via `CustomDebugStringConvertible`. 🙃